.github/workflows/suite-native_staging_ci.yml

```
@@ -48,19 +54,19 @@ jobs:
       - name: Fastlane build and deploy
         working-directory: suite-native/app/
         env:
-          SIGNING_KEY_FILE: stag-upload-key.keystore
-          SIGNING_KEY_STORE_PASSWORD: ${{ secrets.SIGNING_KEY_STORE_PASSWORD_STAGING }}
-          SIGNING_KEY_ALIAS: ${{ secrets.SIGNING_KEY_ALIAS_STAGING }}
-          SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD_STAGING }}
+          SIGNING_KEY_FILE: prod-upload-key.keystore
+          SIGNING_KEY_STORE_PASSWORD: ${{ secrets.SIGNING_KEY_STORE_PASSWORD_PROD }}
+          SIGNING_KEY_ALIAS: ${{ secrets.SIGNING_KEY_ALIAS_PROD }}
+          SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD_PROD }}
           GOOGLE_PLAY_JSON_KEY_FILE: ./android/google_play_key.json
         # builds and publishes the app
         run: |
           bundle install
-          bundle exec fastlane android staging
+          bundle exec fastlane android production
```

```suggestion
          bundle exec fastlane android staging
```

Below, you're running staging lane for iOS, should be same for android